### DISCUSSION TITLE
24/11/04 주간 정기 보고

### DISCUSSION THREAD

**Discussion by gimsoomin** [cloud2scape/.github#24] [Announcements]
### 수민
- [x] 앤서블 기초강의 수강
- [x] aws 모니터링 핸즈온랩 참석

### 재욱
- [x] Organization 구성 및 github 교육
- [x] 프로젝트 패키지 구조 작성
- [x] MSA 프로젝트 생성(3 services + DS + gateway)
- [x] CI(이미지 자동화)
- [x] Account 서비스 스켈레톤 완성

### 수민
- [x] aws 핸즈온랩 완료
- [ ] eks wordpress 올리기
- [x] 깃,깃허브 교육
- [ ] 테라폼으로 시작하는 iac

**Comment by pcj1792** [comment]
## 이번주

### 찬준

- [x] ppt 제작
- [x] 공공데이터 포털에서 데이터 찾기
- [ ] 데이터 가공
- [ ] Loki 설치 후 테스트 해보기

**Comment by ssedogdoom** [comment]
## 이번주

### 나연

- [ ] aws 인강(인프라)
- [ ] 인프라 코드 짜보기(본 아키대로) - yaml file
조정 가능

**Comment by laigasus** [comment]
## 이번주

### 재욱
- [x] 서비스 CRUD 완성
- [x] 통합테스트용 k8s 로컬 구성
- [x] Product 서비스 내 도메인 지정 후 원천 데이터 경로 확보
- [x] API 명세서 보완
- [ ] 서킷 브레이커 구현
- [ ] Replication DB 연결(+CQRS 패턴)
